README.md

```diff
@@ -1,32 +1,32 @@
 <img src="https://static.scarf.sh/a.png?x-pxid=be2d8a11-9712-4c1d-9963-580b2d4fb133" />
 
-![React Query Header](https://github.com/tannerlinsley/react-query/raw/beta/media/repo-header.png)
+![TanStack Query Header](https://github.com/TanStack/query/raw/beta/media/repo-header.png)
 
-Hooks for fetching, caching and updating asynchronous data in React
+Hooks for fetching, caching and updating asynchronous data in React, Solid, Svelte And vue
 
 <a href="https://twitter.com/intent/tweet?button_hashtag=TanStack" target="\_parent">
   <img alt="#TanStack" src="https://img.shields.io/twitter/url?color=%2308a0e9&label=%23TanStack&style=social&url=https%3A%2F%2Ftwitter.com%2Fintent%2Ftweet%3Fbutton_hashtag%3DTanStack">
 </a><a href="https://discord.com/invite/WrRKjPJ" target="\_parent">
   <img alt="" src="https://img.shields.io/badge/Discord-TanStack-%235865F2" />
-</a><a href="https://github.com/tannerlinsley/react-query/actions?query=workflow%3A%22react-query+tests%22">
-<img src="https://github.com/tannerlinsley/react-query/workflows/react-query%20tests/badge.svg" />
-</a><a href="https://npmjs.com/package/react-query" target="\_parent">
-  <img alt="" src="https://img.shields.io/npm/dm/react-query.svg" />
+</a><a href="https://github.com/TanStack/query/actions?query=workflow%3A%22react-query+tests%22">
+<img src="https://github.com/TanStack/query/workflows/react-query%20tests/badge.svg" />
+</a><a href="https://www.npmjs.com/package/@tanstack/react-query" target="\_parent">
+  <img alt="" src="https://img.shields.io/npm/dm/@tanstack/react-query.svg" />
```

Comment:
Maybe this should be kept for now just because the react-query package still has a lot more downloads?

Reply:
Also not sure if it would be good to add shields for the other packages as well or if it would get too messy

Reply:
I think it's fine. The numbers will align sooner or later :)

We can use one shield for the `query-core`, because each adapter implicitly uses that one, and if people do js-only, they will also use the core. The angular adapater (external) also depends on the query-core, so I think it will show the most accurate usage of the lib:

- https://img.shields.io/npm/dm/@tanstack/query-core.svg

numbers are pretty aligned with the ones from react-query, but this might change in the future :)

Reply:
Sure, I can change it to `query-core`. Should I change the bundle size badge too?

Reply:
> Sure, I can change it to query-core

👍 

> Should I change the bundle size badge too?

That's an interesting question. The bundle size for the query-core is smaller because it's missing the adapter specific logic. I wouldn't want that people feel deceived by that...

Reply:
Yeah, fair point. The adapters aren't all the same size, but since it's not a huge difference using the React adapter is probably fine. Especially since it's by far the most used one anyway

Reply:
Updated the downloads badge in https://github.com/TanStack/query/pull/4557/commits/b61010d74030a09aefc2d730643b7e933e731346